### PR TITLE
Add some ideas for what a SIG Update should contain

### PR DESCRIPTION
**Pull request by castrojo**
Fixes for #2395

### CODE REVIEW

File: events/community-meeting.md [hunk 1]
```diff
@@ -63,9 +63,25 @@ Also, if you are doing a live coding demo, please make sure it has a reasonable
...
-SIGs will give a community update at least once per release cycle per the [schedule](https://docs.google.com/spreadsheets/d/1adztrJ05mQ_cjatYSnvyiy85KjuI6-GuXsRsP-T2R3k). 
+## SIG Updates
+
+SIGs will give a community update at least once per release cycle per the [schedule](https://docs.google.com/spreadsheets/d/1adztrJ05mQ_cjatYSnvyiy85KjuI6-GuXsRsP-T2R3k).
+The SIG Update should mention:
+
+- Topics where input is being sought from other SIGs
+- Topics that could affect other SIGs
+- Currently active themes and goals in the SIG
+  - Broad description of future themes and goals if possible
```
Comment (tpepper) [inline events/community-meeting.md]: Was this intentionally indented?  It makes me wonder if this big bullet list might be organized into a couple three or four major topic areas with the detailed bullets under them?  But that's really a nit and can be modified in future if desired.

/lgtm

**Comment by idvoretskyi** [comment]
/hold
for other comments

/lgtm

**Comment by parispittman** [comment]
I’d add update on sponsoring working groups, if applicable

**Pull request merged**